ios/brave-ios/App/l10n/tools/xliff-cleanup.py

```diff
@@ -40,16 +45,18 @@
             for file_node in root.xpath("//x:file", namespaces=NS):
                 original = file_node.get('original')
                 if original and original.endswith('InfoPlist.strings'):
-                    for trans_unit_node in file_node.xpath("./x:body/x:trans-unit", namespaces=NS):
-                        id = trans_unit_node.get('id')
-                        if id and id in STRINGS_TO_REMOVE:
+                    for trans_unit_node in file_node.xpath(
+                            "./x:body/x:trans-unit", namespaces=NS):
+                        string_id = trans_unit_node.get('id')
+                        if string_id and string_id in STRINGS_TO_REMOVE:
                             trans_unit_node.getparent().remove(trans_unit_node)
 
             # 2. Remove empty file sections
             for file_node in root.xpath("//x:file", namespaces=NS):
                 original = file_node.get('original')
                 if original and original.endswith('Info.plist'):
-                    trans_unit_nodes = file_node.xpath("x:body/x:trans-unit", namespaces=NS)
+                    trans_unit_nodes = file_node.xpath("x:body/x:trans-unit",
```

Comment:
are those kind of changes due to brave-core formatter?

Reply:
Yep, `npm run format` applies python formatting